src/target/riscv/riscv.h

```diff
@@ -137,6 +137,8 @@ struct riscv_info {
 	/* The number of triggers per hart. */
 	unsigned int trigger_count;
 
+	struct list_head *wp_triggers_cache;
```

Comment:
This needs a comment explaining what the cache is for. What does it mean for a value to be in the cache?

Also mention that it's an array of caches, one for each trigger.

Reply:
I had several ideas that correspond to Tim's comments but forgot to submit my pending review comments :man_facepalming:  -- I'm sorry.

Here I'd recommend to:

- 1) stress that this is a **negative** cache directly in the name of the variable -- we cache items that are known to **not** work
- 2) add an explanatory comment

For example:

```suggestion
	/* Data structure to record known unsupported tdata1+tdata2 trigger CSR values.
	 * This is to avoid repetitive attempts to set trigger configurations that are already
	 * known to be unsupported in the HW. */
	struct list_head *wp_triggers_negative_cache;
```

or 

```suggestion
	/* Data structure to record known unsupported tdata1+tdata2 trigger CSR values.
	 * This is to avoid repetitive attempts to set trigger configurations that are already
	 * known to be unsupported in the HW. */
	struct list_head *wp_triggers_unsupported_values;
```

or similar

Reply:
Addressed.